src/Utility/RouteGuessingModelResolver.php

```diff
@@ -18,8 +18,6 @@
  */
 class RouteGuessingModelResolver implements ModelResolver
```

Comment:
This resolver is not used in our projects and it had no test coverage.

I assumed that it was meant to work with named routes with a name formatted as `version.resource` since the comments refer to a version and resource.